src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
-body {
+@import "./utils/variables";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+a {
+  display: block;
+  text-decoration: none;
```

Comment:
Same for `text-decoration`, you should only reset this in some CSS classes for elements that you want to, not for global `a` elements in the whole project.